config/resources/press-releases.json

```diff
@@ -15,7 +15,8 @@
     "nie": "http://www.semanticdesktop.org/ontologies/2007/01/19/nie#",
     "skos": "http://www.w3.org/2004/02/skos/core#",
     "vcard": "http://www.w3.org/2006/vcard/ns#",
-    "prov": "http://www.w3.org/ns/prov#"
+    "prov": "http://www.w3.org/ns/prov#",
+    "kans": "http://kanselarij.vo.data.gift/core/"
```

Comment:
Deze prefix wordt niet gebruikt in deze file dus moet hier niet toegevoegd worden.